### DISCUSSION TITLE
how to hide and how the terminal pane?

### DISCUSSION THREAD

**Discussion by c287** [zed-industries/zed#14627] [Feature Requests]
i want to be able to hide and show the terminal pane with a keybind. currently have this config but if i close and reopen the terminal the terminal session isn't saved.
```json
  {
    "context": "Terminal",
    "bindings": {
      "ctrl-/": "pane::CloseActiveItem"
    }
  },
  {
    "context": "VimControl && !menu",
    "bindings": {
      "ctrl-/": "workspace::NewCenterTerminal"
    }
  },
```

**Comment by cs2018ncsa** [comment]
this and keybindings to resize panes would be awesome.

**Comment by wiiznokes** [comment]
This worked for my use case:

```json
[
  {
    "bindings": {
      "ctrl-q": "workspace::ToggleRightDock"
    }
  }
]
```

**Comment by h4ndy-on-github** [comment]
In Zed 0.201.0, the default is **Ctrl + \`** or **Ctrl + J** to open the terminal and **Ctrl + J** to close the terminal.  You can also click **Close Bottom Dock** icon at the left of the status bar to close the terminal while it is opening.

Pressing **Ctrl + \`**  multiple times will switch between editor and terminal.

<img width="1919" height="97" alt="close_bottom_dock_zed" src="https://github.com/user-attachments/assets/8f14f811-7905-4b78-861b-f5bad2707e92" />
